apps/ledger-live-mobile/android/dapp-store/config.yaml

```diff
@@ -0,0 +1,139 @@
+publisher:
+  name: Ledger SAS
+  address: 7hmxf1enboeg7fxWLrMvKRGi9r1NbDG8j7i7MWtAmLLk
+  website: https://www.ledger.com/pages/ledger-live
+  email: live-release-support@ledger.fr
+  media:
+    - purpose: icon
+      uri: ./media/playstore-icon.png
+app:
+  address: 7SE1D8eUsS3yAExzStukTwDDXzcC1xCh2nYX5CFzsMsT
+  name: Ledger Live
+  android_package: com.ledger.live
+  urls:
+    license_url: https://shop.ledger.com/pages/ledger-live-terms-of-use
+    copyright_url: https://shop.ledger.com/pages/ledger-live-terms-of-use
+    privacy_policy_url: https://www.ledger.com/privacy-policy
+    website: https://ledger.com
+  media:
+    - purpose: icon
+      uri: media/playstore-icon.png
+release:
+  address: ""
+  catalog:
+    en-US:
+      name: "Ledger Live: Crypto & NFT App"
+      short_description: Ledger Live Mobile 3.16.0
```

Comment:
It's in the release documentation to update here the `short_description` before each LLM release

Reply:
Okay, we might want to update the confluence page used to track release actions because I don't think they are aware and people tend to follow this instead of the release documentation